src/services/organizeImports.ts

```diff
@@ -699,3 +764,51 @@ function tryGetNamedBindingElements(namedImport: ImportDeclaration) {
         ? namedImport.importClause.namedBindings.elements
         : undefined;
 }
+
+function getOrganizeImportsOrdinalStringComparer(ignoreCase: boolean) {
+    return ignoreCase ? compareStringsCaseInsensitiveEslintCompatible : compareStringsCaseSensitive;
+}
+
+function getOrganizeImportsUnicodeStringComparer(ignoreCase: boolean, preferences: UserPreferences): Comparer<string> {
+    const resolvedLocale = getOrganizeImportsLocale(preferences);
+    const caseFirst = preferences.organizeImportsCaseFirst ?? false;
+    const numeric = preferences.organizeImportsNumericCollation ?? false;

```

Comment:
Ah, yeah, here's the mismatch; the doc comment says `true` is the default, but it's `false` here.

Reply:
I can't say I know what the default _should_ be; I don't think we did this previously so false is probably safer (though reasonably, I think true is "better", but maybe it's just "different").

Reply:
The default for `Intl.Collator` is `false`. I personally thing numeric collation is better, but wanted the default behavior for `organizeImpotsCollation` to match `new Intl.Collator(locale)`.

Reply:
Thanks for fixing the comment.

Reply:
Also, the default for `simple-import-sort` is apparently `{ sensitivity: "base", numeric: true }`, which means you'd need to set the following to match reliably:

```
{
  organizeImportsCollation: "unicode",
  organizeImportsIgnoreCase: true,
  organizeImportsNumericCollation: true,
}
```

Reply:
That's what it does as a first step, but if that comparison returns equal, it falls back to `a < b` i.e. ordinal (see https://github.com/lydell/eslint-plugin-simple-import-sort/#sorting)

These are the settings I've been using that seem to work:

```js
    "typescript.unstable": {
        "organizeImportsCollation": "unicode",
        "organizeImportsCaseFirst": "upper",
        "organizeImportsNumericCollation": true,
    },
```